notebooks/240-named-entity-recognition/240-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>it is tautology in title like coloful color :)</p><p>and also this part except visualization add nothing with previous cell where you show output of pipeline. I think they can be combined in one</p><p>BTW, can we use gradio interface for demo?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
added demo with gradio